example/src/new_api/manualGestures/index.tsx

```diff
@@ -82,7 +82,7 @@ export default function Example() {
       }
     })
     .onTouchesCancelled((e, manager) => {
-      for (const touch of e.changedTouches) {
+      for (const touch of e.allTouches) {
```

Comment:
In case of cancel, wouldn't both arrays contain exactly the same pointers?

Reply:
At first they didn't and that's why I changed that. Then I slightly changed cancel behavior and right now they do contain both. However I left that change to emphasize that we do cancel every pointer